app/src/main/java/net/gsantner/markor/util/ShareUtil.java

```diff
@@ -33,6 +33,10 @@ public ShareUtil(Context context) {
         setChooserTitle(_context.getString(R.string.share_to_arrow));
     }
 
+    Context getContext() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/gsantner/markor/pullRequest?prid=5279643)